language-plutus-core/src/Language/PlutusCore/Lexer.x

```diff
@@ -2,6 +2,7 @@
     {-# OPTIONS_GHC -fno-warn-unused-imports #-}
     {-# LANGUAGE DeriveAnyClass     #-}
     {-# LANGUAGE DeriveGeneric      #-}
+    {-# LANGUAGE DeriveLift         #-}
```

Comment:
If possible, add `DeriveLift` to the `other-extensions` field of the `.cabal` file.

Reply:
:+1: